doc/source/indexing.rst

```diff
@@ -370,6 +373,15 @@ Accessing via label slices
 
    df1.loc['d':, 'A':'C']
 
+.. warning::
+
+   If both the start and the stop labels are present in the index, then
+   elements *located* between the two (including them) are returned. If at
+   least one of the two is absent but the index is sorted (and can be compared
+   against start and stop labels), then slicing will still work, by selecting
```

Comment:
ok, I think we need a mini-example here, this text is pretty dense. can you make a sub-section with a couple of examples. 

This is just not obvious to new users (whom we presume read the documentation). want to make it really clear what is going on. The examples from the issue are pretty good and show the situation.